proprietary/tokens/src/components/ams/grid.tokens.json

```diff
@@ -2,17 +2,17 @@
   "ams": {
     "grid": {
       "column-count": { "value": "4" },
-      "column-gap": { "value": "{ams.space.grid.md}" },
+      "column-gap": { "value": "{ams.space.xl}" },
       "padding-block": {
-        "sm": { "value": "{ams.space.grid.sm}" },
-        "md": { "value": "{ams.space.grid.md}" },
-        "lg": { "value": "{ams.space.grid.lg}" }
+        "sm": { "value": "{ams.space.md-lg}" },
+        "md": { "value": "{ams.space.lg-xl}" },
+        "lg": { "value": "{ams.space.xl-2xl}" }
```

Comment:
I’m not sure we still need these three options – 50%, 100% or 150% of a grid gap as padding above or below it. I think we can just keep the medium value, like we do for horizontal (inline) padding.

If people want more than a grid gap of whitespace below something, they can add a bottom margin through or `.ams-mb-*` utility classes.